pkg/apis/pipeline/v1/container_types.go

```diff
@@ -188,6 +188,35 @@ func (s *Step) SetContainerFields(c corev1.Container) {
 	s.SecurityContext = c.SecurityContext
 }
 
+// GetVarSubstitutionExpressions walks all the places a substitution reference can be used
+func (s *Step) GetVarSubstitutionExpressions() []string {
```

Comment:
please add tests for this exported function 

https://github.com/tektoncd/community/blob/main/standards.md#go-packages